libfreerdp/core/info.c

```diff
@@ -73,14 +73,18 @@ static const struct info_flags_t info_flags[] = {
 	{ INFO_HIDEF_RAIL_SUPPORTED, "INFO_HIDEF_RAIL_SUPPORTED" },
 };
 
-static BOOL rdp_read_info_null_string(const char* what, UINT32 flags, wStream* s, size_t cbLen,
-                                      CHAR** dst, size_t max, BOOL isNullTerminated)
+static BOOL rdp_read_info_null_string(rdpSettings* settings, size_t id, const char* what,
+                                      UINT32 flags, wStream* s, size_t cbLen, size_t max,
+                                      BOOL isNullTerminated)
 {
 	CHAR* ret = NULL;
```

Comment:
Actually a small nit: This variable is now unused.

Reply:
true.